torch/quantization/fx/utils.py

```diff
@@ -74,3 +74,58 @@ def graph_pretty_str(g, shorten=True) -> str:
     if shorten:
         res_str += "*obs_{n} = activation_post_process_{n}\n"
     return res_str
+
+def is_per_tensor(qscheme):
+    return qscheme == torch.per_tensor_affine or \
+        qscheme == torch.per_tensor_symmetric
+
+def is_per_channel(qscheme):
+    return qscheme == torch.per_channel_affine or \
+        qscheme == torch.per_channel_symmetric
```

Comment:
Should we also include `per_channel_affine_float_qparams` here?

Reply:
yeah i think so, will add